example/test_problem/Hydro/CCSN/Input__TestProb.Leakage

```diff
@@ -0,0 +1,40 @@
+# problem-specific runtime parameters
+CCSN_Prob        2                                 # target CCSN problem
+                                                   #    0 : GREP migration test
+                                                   #    1 : Post bounce test
+                                                   #    2 : Core collapse test
+
+CCSN_Prof_File   IC/s20_W2007                      # filename of input profile
+
+CCSN_Mag         1                                 # target magnetic field profile                     [1]
+                                                   # ( 0: Ap = B0* varpi^2 * (1 - rho / rho_max)^np * (P / P_max)
+                                                   #   1: Ap = 0.5 * B0 * ( R0^3 / (r^3 + R0^3) ) * r * sin(theta) )
+CCSN_Mag_B0      1.0e14                            # magnetic field strength (in gauss)                [1.e14]
```

Comment:
default B0 better set to 0?

Reply:
True. will be updated in next PR for arranging the CCSN test problem.